Use clone depth of 1 to speed up build times

nurviro opened the pull request:
In the Docker build instructions, might as well use a clone depth of 1 to minimize the time it takes to build.

nurviro committed:
Use clone depth of 1 to speed up build times

In the Docker build instructions, might as well use a clone depth of 1 to minimize the time it takes to build.

knaccc commented:
Thanks, are you able to modify your pull request so that it does not touch the last line please? That's quite a sensitive line to see a change on, even though it appears it is probably nothing more than a CR/LF issue.

nurviro commented:
Thanks, fixed it and squashed the commits. That's what I get for trying the built-in Github editor I guess...

On a similar note, do you mind if I submit a PR to use `--depth 1` for git in the build scripts? (Like cloning i2p in the `import-packages.sh`)

knaccc commented:
@nurviro it's a good idea, the following could be done:

`git clone --branch tags/i2p-0.9.48 --depth 1 https://github.com/i2p/i2p.i2p.git`

The problem is I've not figured out how to do a 'git branch' on a shallow clone to see what tag has been checked out, so that I can make the script delete and re-clone if the stored clone is not the same as the tag that was previously checked out. Any ideas on how to do that?

knaccc commented:
@nurviro it's a good idea, I've just pushed the change
